### PR TITLE
Add Mesos-Marathon as a swarm frontend

### PR DESCRIPTION
**Pull request by amitshukla**
Adding a Compose file that uses Swarm to deploy Mesos-marathon to a swarm cluster. I reorganized the root README.md file so works with having multiple swarm frontends.

### CODE REVIEW

File: mesos-marathon/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+New nodes show up in the Mesos dashboard as expected.
+
+## Known limits / Possible enhancements
+
+This is a base Compose file but it can be used to tweak a complete Production deployment, securing it with TLS, using distributed volumes (Ceph, GlusterFS) for the etcd cluster, etc. Feel free to propose enhancements! We'd like to see it working better.
+
+A needed enhancement on the Swarm side is the use of ACLs to deploy the Mesos-Marathon cluster. This allows an admin user to deploy the cluster and regular users will not be able to delete the Mesos-Marathon components (It's in the work in [Swarm #1366](https://github.com/docker/swarm/pull/1366))
+
+
+@Vieux are there other known limitations?
```
Comment (abronan) [inline mesos-marathon/README.md]: We can maybe remove that :smile:

Comment (abronan) [inline mesos-marathon/README.md]: @vieux any limitation on running Mesos on top of Swarm that you may be aware of? (see comment above).

**Comment by abronan** [comment]
Good call on the startup script, overall LGTM.

File: mesos-marathon/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+To scale mesos-slave instances you can now use:
+
+`docker-compose --x-networking -f mesos-marathon-swarm.yml scale messoslave=4`
+
+New nodes show up in the Mesos dashboard as expected.
+
+## Known limits / Possible enhancements
+
+This is a base Compose file but it can be used to tweak a complete Production deployment, securing it with TLS, using distributed volumes (Ceph, GlusterFS) for the etcd cluster, etc. Feel free to propose enhancements! We'd like to see it working better.
```
Comment (abronan) [inline mesos-marathon/README.md]: you might want to change: `etcd` -> `zookeeper`

**Comment by amitshukla** [comment]
@abronan I verified that the init_swarm script works with kubernetes as well. Can you please take another look and if it looks good, merge it in? Thanks.

**Comment by abronan** [comment]
Thanks for taking care of the comments @amitshukla,

LGTM

**Pull request merged**